src/ble/CHIPBleServiceData.h

```diff
@@ -64,17 +66,31 @@ struct ChipBLEDeviceIdentificationInfo
 
     void SetProductId(uint16_t productId) { chip::Encoding::LittleEndian::Put16(DeviceProductId, productId); }
 
+    uint16_t GetAdvertisementVersion() const
+    {
+        return chip::Encoding::LittleEndian::Get16(DeviceDiscriminatorAndAdvVersion) & kAdvertisementMask;
```

Comment:
So if I do `SetAdvertisementVersion(1)` and then `GetAdvertisementVersion()`... what value will I get?

Reply:
Now it will return actual advertisement version 1